webapi/src/test/scala/org/knora/webapi/e2e/admin/lists/OldListsRouteADMFeatureE2ESpec.scala

```diff
@@ -1038,7 +1038,7 @@ class OldListsRouteADMFeatureE2ESpec
           TestDataFileContent(
```

Comment:
Do we still need this whole OldList... things?

Reply:
Yes, this is the only one listsADM implementation, the "new" one was deleted.

Reply:
Ah, OK. Shouldn't we rename it then to just "List..." instead of "OldList..."?